src/components/Bounties/ClaimChildBountyButton.tsx

```
@@ -0,0 +1,156 @@
+	const { loginAddress, loginWallet, multisigAssociatedAddress } = useUserDetailsSelector();
+	const {
+		postData: { status, curator, postIndex }
+	} = usePostDataContext();
+	const [loading, setLoading] = useState<boolean>(false);
+	const [multisigData, setMultisigData] = useState<{ threshold: number; signatories: string[] }>({
+		signatories: [],
+		threshold: 0
+	});
+
+	const getOnChainTx = async () => {
+		if (!api || !apiReady || bountyIndex == null || isNaN(bountyIndex)) return null;
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)